iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -199,6 +206,18 @@ void buildFlowTransformPassPipeline(OpPassManager &passManager) {
   // Symbol DCE any remaining variables/functions that are now no longer
   // required.
   passManager.addPass(createSymbolDCEPass());
+
+  // Export all the dispatch functions. This creates functions and feeds
+  // iree.unfoldable_constant as inputs to the exported functions.
+  if (exportDispatchFunctions) {
+    passManager.addPass(IREE::Flow::createCreateFuncsToInvokeExecOpsPass());
+    passManager.addPass(IREE::Flow::createMaterializeExportedReflection());
```

Comment:
I assume these passes are NOP for main exported function ?

Reply:
Yes, you're right.